packages/cozy-harvest-lib/src/components/KonnectorModalHeader.jsx

```diff
@@ -2,16 +2,28 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { ModalHeader } from 'cozy-ui/transpiled/react/Modal'
 import KonnectorIcon from './KonnectorIcon'
+import cx from 'classnames'
 
-const KonnectorModalHeader = ({ konnector, children }) => {
+const KonnectorModalHeader = ({ konnector, title, vertical, children }) => {
   return (
     <ModalHeader className="u-pr-2">
-      <div className="u-flex u-flex-row u-w-100 u-flex-items-center">
-        <div className="u-w-3 u-h-3 u-mr-half">
+      <div
+        className={cx(
+          'u-w-100',
+          'u-flex',
+          'u-flex-items-center',
+          vertical ? 'u-flex-column' : 'u-flex-row'
+        )}
+      >
+        <div
+          className={cx('u-w-3', 'u-h-3', vertical ? 'u-mb-half' : 'u-mr-half')}
+        >
           <KonnectorIcon konnector={konnector} />
         </div>
-        <div className="u-flex-grow-1 u-mr-half">
-          <h3 className="u-title-h3 u-m-0">{konnector.name}</h3>
+        <div
+          className={cx('u-flex-grow-1', vertical ? 'u-mb-half' : 'u-mr-half')}
+        >
+          <h3 className="u-title-h3 u-m-0">{title}</h3>
```

Comment:
IMHO having to introduce a new API (`vertical`) and having to conditionally put utility classes shows that we are not working at the right level here.

What we seem to want here seems to switch between a stack based layout and a row based layout, with margins between items :

```
const Wrapper = this.props.wrapper // (Stack || Row)
<Wrapper spacing='m'>
</Wrapper>
```

Reply:
What happened to this ?

Reply:
Wtf, I thought I posted a comment explaining the latest changes. Must have forgotten to send it..? Adding it again.